### PR TITLE
Improve markdown file

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/design/features/crossgen2-compilation-structure-enhancements.md [hunk 1]
```diff
@@ -37,27 +39,29 @@ The default value of the CompilationVersionBubble flag would be dependent on how
...
-2. The set of files that comprise the set of assemblies being compiled at once. These are specified via the --input-file-path switch. (The first assembly specified in this list describes module actually to be compiled and output. When we add support for composite R2R images, we will produce a single module file from all of these input assemblies at once.)
-3. The set of modules that comprise the version bubble tied to the assembly being compiled. The specification of this is complicated and unclear at this time. My current idea is to have two modes. Input bubble mode (where the set of input files matches the version bubble), and all mode (where all assemblies are considered to be part of the bubble. For framework ahead of time compilation scenarios such as containers and such where we may be compiling the framework and supporting scenarios where not all files in the framework are necessarily unified in the bubble, we may need an exclude function to exclude specific binaries.
+- The set of files that are referenceable by the compilation. These are specified via the --reference switch.
```
Comment (stephentoub) [inline docs/design/features/crossgen2-compilation-structure-enhancements.md]: Why was the numbered list problematic?

File: docs/design/features/crossgen2-compilation-structure-enhancements.md [hunk 2]
```diff
@@ -87,13 +92,16 @@ I propose to reduce the generics duplication problem to allow duplication betwee
...
 3.  Allows use of proven technology (profile guided optimization) to drive generics compilation for high value customers, while also provided an opportunity for innovation in heuristics.
 
 ## Issues to resolve with this proposal
+
 1. This proposal needs to be squared with our servicing policies and ensure that we aren't affecting preventing promised servicing capabilities.
-  - In particular, there has been a concept in the past about making self contained applications serviceable through a machine wide policy key. We would need to design what our plans are here.
\ No newline at end of file
+  - In particular, there has been a concept in the past about making self contained applications serviceable through a machine wide policy key. We would need to design what our plans are here.
```
Comment (stephentoub) [inline docs/design/features/crossgen2-compilation-structure-enhancements.md]: What changed here?

Comment (Youssef1313) [reply docs/design/features/crossgen2-compilation-structure-enhancements.md]: I didn't touch this line myself. But I *think* GitHub added a new line at the end of the file to comfort to [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character).

Comment (Youssef1313) [reply docs/design/features/crossgen2-compilation-structure-enhancements.md]: I've confirmed that GitHub added it automatically by testing on another branch in my fork.

File: docs/design/features/crossgen2-compilation-structure-enhancements.md (hunk 1 above)
Comment (stephentoub) [reply docs/design/features/crossgen2-compilation-structure-enhancements.md]: It says that there are three things and then lists the three things; numbers make sense there... it's not about order but about counting.

Comment (Youssef1313) [reply docs/design/features/crossgen2-compilation-structure-enhancements.md]: Reverted those.

**Review by stephentoub — APPROVED** [review]
Thanks.

**Pull request merged**